laravel/resources/views/administration/users.blade.php

```diff
@@ -1,173 +1,121 @@
-                                <p class="card-text">
-                                    <ul>
-                                        @if ($role->name == "Super Admin")
-                                            <li>All permissions</li>
-                                        @else
-                                            @foreach ($role->permissions->pluck('name') as $permission)
-                                                <li>{{ $permission }}</li>
-                                            @endforeach
-                                        @endif
-                                    </ul>
-                                </p>
-                            </div>
-                        </div>
-                    @if (($role_counter % 3) == 2)
-                    </div>
-                    @endif
-
-                    @php
-                        ++$role_counter;
-                    @endphp
+<div class="container">
+    <div class="row">
+        <div class="col-lg-3">
+            <button type="button" class="btn btn-primary btn-sm fw-bold" data-bs-toggle="modal" data-bs-target="#newUser">
+                New User
+            </button>
+        </div>
+    </div>
+    <hr>
+</div>
+<div class="container mt-3">
+    @include('inc.standard-alerts')
+    <div class="row">
+        <div class="col-lg-12">
+            <table class="table table-striped" id="roles">
+                <thead>
+                <tr>
+                    <th scope="col">Name</th>
+                    <th scope="col">E-Mail</th>
+                    <th scope="col">Roles</th>
+                    <th scope="col">Registered since</th>
+                    <th scope="col">Actions</th>
+                </tr>
+                </thead>
+                <tbody>
+                @foreach($users as $user)
+                <tr>
+                    <td class="col-lg-2">
+                        {{ $user->name }}
+                    </td>
+                    <td class="col-lg-2">
+                        {{ $user->email }}
+                    </td>
+                    <td class="col-lg-4">
+                        @foreach($user->getRoleNames() as $role)
+                            @switch(true)
+                                @case($role == 'Super Admin')
+                                    <span class="badge text-bg-danger">{{$role}}</span>
+                                @break
+                                @case(preg_match('/Admin/i', $role))
+                                    <span class="badge text-bg-warning">{{$role}}</span>
+                                @break
+                                @default
+                                    <span class="badge text-bg-primary">{{$role}}</span>
+                            @endswitch
+                        @endforeach
+                    </td>
+                    <td class="col-lg-2">
+                        {{date('d.m.Y',strtotime($user->created_at))}}
```

Comment:
we musst use the full path to Carbon `\Illuminate\Support\Carbon::parse($user->created_at)->format('d.m.Y')`
because `\Carbon\Carbon::` is a other function